tests/test_github_issue1062/Snakefile

```diff
@@ -0,0 +1,18 @@
+from snakemake.remote.FTP import RemoteProvider as FTPRemoteProvider    
+FTP = FTPRemoteProvider()    
+    
+rule all:    
+    input:    
+        "robots.txt"
+
+FTP_URL = \
+    "ftp://ftp.ensembl.org/pub/release-100/fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna_sm.primary_assembly.fa.gz"
```

Comment:
Thanks for tracking this down and getting a fix in so quickly. Just a note that the file here is a bit large, you could try something like `ftp://ftp.ensembl.org/pub/release-100/fasta/homo_sapiens/dna/README` instead.

Reply:
Nice catch. I meant to use the robots.tx file. It's smaller and less likely to be removed. I was just switching back and forth between it and the file from the bug report...and I checked in the wrong one.